pkg/gce-pd-csi-driver/controller_test.go

```diff
@@ -136,6 +136,65 @@ func TestCreateVolumeArguments(t *testing.T) {
 				Attributes:    nil,
 			},
 		},
+		{
+			name: "success with topology",
+			req: &csi.CreateVolumeRequest{
+				Name:               "test-vol",
+				CapacityRange:      stdCapRange,
+				VolumeCapabilities: stdVolCap,
+				Parameters:         map[string]string{"type": "test-type"},
+				AccessibilityRequirements: &csi.TopologyRequirement{
+					Requisite: []*csi.Topology{
+						{
+							Segments: map[string]string{"zone": "topology-zone"},
+						},
+					},
+				},
+			},
+			expVol: &csi.Volume{
+				CapacityBytes: utils.GbToBytes(20),
+				Id:            "topology-zone/test-vol",
+				Attributes:    nil,
+			},
+		},
+		{
+			name: "success with topology overwrite zone in params",
```

Comment:
I think we actually wanted to error in this case because we don't just want to drop/ignore user config